queries.sql

```diff
@@ -1,3 +1,60 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+vet_clinic=# SELECT * FROM animals  WHERE NAME LIKE '%mon';
+ id |  name   | date_of_birth | escape_attempts | neutered | weight_kg
+----+---------+---------------+-----------------+----------+-----------
+  1 | Agumon  | 2020-02-03    |               0 | t        |     10.23
+  2 | Gabumon | 2018-11-15    |               2 | t        |      8.00
+  4 | Devimon | 2017-05-12    |               5 | t        |     11.00
+(3 filas)
+
+vet_clinic=# SELECT NAME FROM animals WHERE DATE_OF_BIRTH BETWEEN '2016-01-01' AND '2019-12-31';
+  name
+---------
+ Gabumon
+ Devimon
+(2 filas)
+
+vet_clinic=# SELECT NAME FROM animals WHERE NEUTERED = true AND escape_attempts < 3;
+  name
+---------
+ Agumon
+ Gabumon
+(2 filas)
+
+vet_clinic=# SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
```

Comment:
- [ ] Please remove the results of the queries from here, You already added screenshots of these results in the PR description. Kindly only leave the query itself here because this is the `queries.sql` file meant for.

[Hint] It is supposed to be like that:

```sql
SELECT * FROM animals  WHERE NAME LIKE '%mon'

SELECT NAME FROM animals WHERE DATE_OF_BIRTH BETWEEN '2016-01-01' AND '2019-12-31';

SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');

------------------
------------------
```